docs/standalone.md

```diff
@@ -104,17 +104,16 @@ I0911 13:42:03.657918   21041 heapster.go:58] Heapster version 0.17.0
 I0911 13:42:03.662594   21041 heapster.go:68] Starting heapster on port 8082
 ```
 
-###Test it out
+
+### Test it out
 If you got debug information as before, means that the heapster is working normally.You can test it by calling its [RESTful API](https://github.com/kubernetes/heapster/blob/master/docs/model.md).  
 You can use some restful tools to test, or just run a `curl` command (_ heapster implements gzip encoding, so we need add gunzip in command "curl"_).  
 
-```shell
-curl http://0.0.0.0:8082/api/v1/model/stats/ | gunzip
-```
+	curl http://0.0.0.0:8082/api/v1/model/stats/ | gunzip
```

Comment:
So does heapster compress every response, even if the client hasn't requested compression?